packages/next-swc/crates/core/tests/fixture/styled-jsx/attribute-generation-classname-rewriting/output.js

```diff
@@ -2,62 +2,111 @@ import _JSXStyle from "styled-jsx/style";
+
       <div {...props} className={"jsx-abb4c2202db1a207" + " " + (props && props.className != null && props.className || test && test("test") && "test" || "")}/>
+
       <div {...props} className={"jsx-abb4c2202db1a207" + " " + (props && props.className != null && props.className || test && test("test") && test2("test") || "")}/>
+
       <div {...props} className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <div {...props} {...rest} className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <div {...props} {...rest} className={"jsx-abb4c2202db1a207" + " " + (rest && rest.className != null && rest.className || "test")}/>
+
       <div {...props} className={"jsx-abb4c2202db1a207" + " " + (props && props.className != null && props.className || "")}/>
+
       <div {...props} {...rest} className={"jsx-abb4c2202db1a207" + " " + (rest && rest.className != null && rest.className || props && props.className != null && props.className || "")}/>
+
       <div {...props} data-foo {...rest} className={"jsx-abb4c2202db1a207" + " " + (rest && rest.className != null && rest.className || props && props.className != null && props.className || "")}/>
+
       <div {...props} data-foo {...rest} className={"jsx-abb4c2202db1a207" + " " + (rest && rest.className != null && rest.className || "test")}/>
+
       <div {...{
         id: "foo"
     }} className={"jsx-abb4c2202db1a207"}/>
+
       <div className={"jsx-abb4c2202db1a207" + " " + "foo"}/>
+
       <div {...{
         className: "foo"
     }} className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <div className="test" className={"jsx-abb4c2202db1a207" + " " + "foo"}/>
+
       <div {...bar} className={"jsx-abb4c2202db1a207" + " " + (bar && bar.className != null && bar.className || "foo")}/>
+
       <div {...{
         className: "foo"
     }} {...bar} className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <div className="test" {...bar} className={"jsx-abb4c2202db1a207" + " " + (bar && bar.className != null && bar.className || "foo")}/>
+
       <div className="test" className={"jsx-abb4c2202db1a207" + " " + (props.className || "")}/>
+
       <div className="test" {...bar} className={"jsx-abb4c2202db1a207" + " " + (bar && bar.className != null && bar.className || props.className || "")}/>
+
       <div className="test" {...bar} className={"jsx-abb4c2202db1a207" + " " + (props.className || "")}/>
+
       <div {...bar()} className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <Element className={"jsx-abb4c2202db1a207"}/>
+
       <Element className={"jsx-abb4c2202db1a207" + " " + "test"}/>
+
       <Element {...props} className={"jsx-abb4c2202db1a207" + " " + (props && props.className != null && props.className || "")}/>
-      <_JSXStyle id={"abb4c2202db1a207"}>{"div.jsx-abb4c2202db1a207 {color:red}"}</_JSXStyle>
+
+      <_JSXStyle id={"abb4c2202db1a207"}>{"div.jsx-abb4c2202db1a207{color:red}"}</_JSXStyle>
```

Comment:
I removed a space between the selectors and the block.
Most css changes of this PR is due to removal of the space.